.env.example

```diff
@@ -1,3 +1,3 @@
-NEXT_PUBLIC_SUPABASE_URL=<SUBSTITUTE_SUPABASE_URL>
+NEXT_PUBLIC_SUPABASE_DATABASE_URL=<SUBSTITUTE_SUPABASE_URL>
 NEXT_PUBLIC_SUPABASE_ANON_KEY=<SUBSTITUTE_SUPABASE_ANON_KEY>
```

Comment:
Since we don't use any of these env vars on the client, let's drop the `NEXT_PUBLIC_` prefix to make sure Next doesn't include them in the bundle.